File: practice/My project/randomquestions.py
import time
import random

start_time = time.time()

correct_answer = 0
wrong_answer = 0
right_answer = 0

userinput = int(input("How many questions do you want to answer? "))

for questions in range(userinput):
	question1 = random.randint(1, 100)
	question2 = random.randint(1, 100)

	if question1 > question2:
		temp = question1
		question1 = question2
		question2 = question1
		
		right_answer = question2 + question1
	
	answer = int(input(f"Answer This question {question2} + {question1}? "))

	end_time = time.time()

	if answer == right_answer:
		correct_answer += 1
		print("You  are correct. The answer is = ", right_answer)

	if answer != right_answer:
		wrong_answer += 1
		print("You  are wrong. The answer is = ", right_answer)

	total_time_used = end_time - start_time

print(f"Total Time used on test:  {total_time_used:,.2f}seconds")
print("Correct answers: ", correct_answer)
print("Wrong answers: ", wrong_answer)

File: practice/My project/Chapter5/RandomQuestions.java
public class RandomQuestions{

   public static void main(String [] args){

	java.util.Scanner sc = new java.util.Scanner(System.in);

	long startTime = System.currentTimeMillis()/1000;
	
	long totalTestTime = 0;	

	int correctAnswer = 0;

	int failedAnswer = 0;
	
	System.out.print("How many questions do you want to answer? ");
	int QUESTION_TO_ASK = sc.nextInt();

	for(int question = 1; question <= QUESTION_TO_ASK; question++){

		int questionNumber1 = (int)(Math.random() * 100);
		int questionNumber2 = (int)(Math.random() * 100);

		if(questionNumber1 > questionNumber2){
			int swap = questionNumber1;
			questionNumber1 = questionNumber2;
			questionNumber2 = swap;

			System.out.print("Answer This question" + ">>> " + questionNumber2 + " + " + questionNumber1 + " : ");
			int userinput = sc.nextInt();

			int answer = questionNumber2 + questionNumber1;

			long endTime = System.currentTimeMillis()/1000;
			long testTime = endTime - startTime;

				totalTestTime += testTime;
			
			if(userinput == answer){
				correctAnswer++;
				System.out.println("You are correct.");
			}

			if(userinput != answer){
				failedAnswer++;
				System.out.println("You are wrong. The answer is = " + answer);
			}

			System.out.println("Time used on test: " + testTime + "seconds");
			System.out.println();
		}	
	}

	System.out.println("Total time used on all test: " + totalTestTime + "seconds");

	System.out.println("Correct answers: " + correctAnswer);
	System.out.print("Failed Answers: " + failedAnswer);
   }

}